server/src/main/java/io/unitycatalog/server/persist/CatalogRepository.java

```diff
@@ -115,7 +115,12 @@ public CatalogInfoDAO getCatalogDAO(Session session, String name) {
     }
 
     public CatalogInfo updateCatalog(String name, UpdateCatalog updateCatalog) {
-        ValidationUtils.validateSqlObjectName(updateCatalog.getNewName());
+        if (updateCatalog.getNewName() == null && updateCatalog.getComment() == null) {
```

Comment:
Instead of throwing an error, the update should be a no-op with success if neither fields are specified. This will make adding new updatable fields easier later.

Reply:
Ok, done. I changed this to simply return the existing catalog info instead of throwing.

With this change, the CLI probably also should not error out. I reverted the previous change to how CliException is handled so the CLI now just prints the info message about update parameters without an error.

Also updated tests to test that an empty update does not change anything instead of testing that an exception is thrown.